sdk/nodejs/src/client.ts

```diff
@@ -175,6 +167,19 @@ export class DevCycleClient<
         })
     }
 
+    private setPlatformDataInBucketingLib(): void {
+        if (!this.bucketingLib) return
+
+        const platformData: IPlatformData = {
+            platform: this.openFeatureProvider ? 'NodeJS-OF' : 'NodeJS',
+            platformVersion: process.version,
+            sdkType: 'server',
+            sdkVersion: packageJson.version,
```

Comment:
do you want to include the openfeature sdk version here too as a customdata or something?